layouts/_default/baseof.html

```diff
@@ -11,9 +11,9 @@
   <div class="container-fluid">
         <div class="row">
           {{ if .IsPage }}
+            {{ $path := split $.Source.File.Path .Site.Params.pathseperator }}
+            {{ if or (eq (index $path 0) "events") (eq .Type "events") }}
 
-            {{ $url_path := split $.Permalink "/" }}
```

Comment:
The problem has to do with how permalinks get created based upon baseurl, etc, so it was splitting weird. We can't trust permalinks.